recipes/vcflib/build.sh

```diff
@@ -26,13 +28,9 @@ if [ "$(uname)" == "Darwin" ]; then
     sed -i.bak 's/if ( n_data/if ( \*n_data/' src/cdflib.cpp
     
 fi
-# tabix missing library https://github.com/ekg/tabixpp/issues/5
-# Uses newline trick for OSX from: http://stackoverflow.com/a/24299845/252589
-#sed -i.bak 's/SUBDIRS=./SUBDIRS=.\'$'\n''LOBJS=tabix.o/' tabixpp/Makefile
-#sed -i.bak 's/-ltabix//' Makefile
-#sed -i.bak 's/make/make -e/' Makefile
 
-make -e
+# -e lets our exported CXX etc override any Makefile settings
+make --environment-overrides -j${CPU_COUNT}
```

Comment:
That's a much nicer solution :)

Reply:
I think I accidently removed `make -e` when I was updating the formula! Whoops :)

Reply:
Still more to do though. It's a mess.